### PR TITLE
Leftpush

### PR DESCRIPTION
**Pull request by nikkobautista**

### CODE REVIEW

**Comment by nikkobautista** [comment]
Sorry for the spacing diff, I tried to convert all my changes from spaces-tab but it also changes some of the spacing issues already present. https://github.com/brightjobs/php-resque/pull/1/files?w=1 will show just my actual changes.

**Comment by gwintrob** [comment]
push it

File: lib/Resque/Job.php [hunk 1]
```diff
@@ -58,11 +58,20 @@ public static function create($queue, $class, $args = null, $monitor = false)
...
+				'class'	=> $class,
+				'args'	=> array($args),
+				'id'	=> $id,
+			));
+		} else {
+			Resque::rpush($queue, array(
+				'class'	=> $class,
+				'args'	=> array($args),
+				'id'	=> $id,
+			));
```
Comment (cecchi) [inline lib/Resque/Job.php]: Nit-picky, but I'd make the second param a variable so you don't repeat yourself:

```php
$item = array(
    'class'	=> $class,
    'args'	=> array($args),
    'id'	=> $id,
);

if($lpush) {
    Resque::lpush($queue, $item);
} else {
    Resque::rpush($queue, $item);
}
```

Comment (nikkobautista) [inline lib/Resque/Job.php]: Yeah, that looks much better. 1 min

**Comment by cecchi** [comment]
:zap:

**Comment by cecchi** [comment]
I lied, fix the indentation please :heart:

File: lib/Resque/Job.php [hunk 2]
```diff
@@ -58,11 +58,18 @@ public static function create($queue, $class, $args = null, $monitor = false)
...
-			'class'	=> $class,
-			'args'	=> array($args),
-			'id'	=> $id,
-		));
+
+        $item = array(
+            'class'	=> $class,
+            'args'	=> array($args),
+            'id'	=> $id,
+        );
```
Comment (cecchi) [inline lib/Resque/Job.php]: I messed up your indentation

**Comment by cecchi** [comment]
:pushpin:

**Pull request merged**